nova/tests/unit/api/openstack/compute/test_volumes.py

```diff
@@ -1234,25 +1260,50 @@ def test_update_volume(self, mock_bdm_save,
             device_name='/dev/fake0',
             attachment_id=uuids.attachment_id)
         mock_get_vol_and_inst.return_value = vol_bdm
-
         body = {'volumeAttachment': {
             'volumeId': FAKE_UUID_A,
             'tag': 'fake-tag',
             'delete_on_termination': True,
             'device': '/dev/fake0',
         }}
+        conn_info = {'driver_volume_type': 'vmdk',
+                     'volume_id': FAKE_UUID_A,
+                     'attachment_id': uuids.attachment_id,
+                     'data':
+                     {
+                        'volume_id': FAKE_UUID_A,
+                        'name': FAKE_UUID_A,
+                        'profile_id': FAKE_UUID_C,
+                        'datastore': 'datastore-1839585',
+                        'qos_specs': None,
+                        'access_mode': 'rw',
+                        'encrypted': False,
+                        'cacheable': False
+                     }
+                     }
```

Comment:
Really sorry, this is really the last nitpick, but the indentation is super weird here. Could you change it to this?

```suggestion
        conn_info = {
            'driver_volume_type': 'vmdk',
            'volume_id': FAKE_UUID_A,
            'attachment_id': uuids.attachment_id,
            'data': {
                'volume_id': FAKE_UUID_A,
                'name': FAKE_UUID_A,
                'profile_id': FAKE_UUID_C,
                'datastore': 'datastore-1839585',
                'qos_specs': None,
                'access_mode': 'rw',
                'encrypted': False,
                'cacheable': False
            }
        }
```